src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -354,6 +355,10 @@ def acr_task_run(cmd,
 
     run_id = queued_run.run_id
     logger.warning("Queued a run with run ID: %s", run_id)
+
+    if no_wait:
```

Comment:
do we need to support this in `build.py` and `run.py` too?

Reply:
I guess we don't need for build.py,